# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/ProtoKey.h
# Commit: 23cc649c6e14
# Full Hash: 23cc649c6e14647072116e6d2396cb615996aa61
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 1
# Description:
#   Bug 1720422 - Move js::Scalar -> JS::Scalar for public API, but alias it as js::Scalar r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120292
# ==============================================================================

diff -r 16695064a637 -r 23cc649c6e14 js/public/ProtoKey.h
--- a/js/public/ProtoKey.h	Thu Aug 12 20:44:17 2021 +0000
+++ b/js/public/ProtoKey.h	Thu Aug 12 20:44:18 2021 +0000
@@ -31,7 +31,7 @@
 
 #define CLASP(NAME) (&NAME##Class)
 #define OCLASP(NAME) (&NAME##Object::class_)
-#define TYPED_ARRAY_CLASP(TYPE) (&TypedArrayObject::classes[Scalar::TYPE])
+#define TYPED_ARRAY_CLASP(TYPE) (&TypedArrayObject::classes[JS::Scalar::TYPE])
 #define ERROR_CLASP(TYPE) (&ErrorObject::classes[TYPE])
 
 #ifdef JS_HAS_INTL_API